### PR TITLE
C.H.I.P. and Omega2 examples

### PR DESCRIPTION
**Pull request by Himura2la**
- [x] Test on C.H.I.P.
- [x] Test on Omega2
- [ ] Review

### CODE REVIEW

File: omega2/omega2.py [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+        Pass a boolean argument to turn the Omega2 LED on and off.
+        Pass a string to trigger the LED mode.
+        Available modes: 'none', 'mmc0', 'timer', 'default-on', 'netdev',
+                         'transient', 'gpio', 'heartbeat', 'morse', 'oneshot'.
+            - When switching to the 'timer' mode, pass the *delay_on* and
+                *delay_off* arguments.
+            - When switching to the 'morse' mode, pass the *message* argument.
+        Details: https://docs.onion.io/omega2-docs/the-omega-led.html
+        """
+        if type(state) in {bool, int}:  # pylint: disable=C0123
```
Comment (sky3d) [inline omega2/omega2.py:39]: why not to use isinstance() ?

File: chip/gpio.py [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+
+from os import uname
+from socket import gethostname
+from time import sleep
+import sys
+import cloud4rpi
+import chip
+
+# Get the GPIO module here: https://github.com/xtacocorex/CHIP_IO
+import CHIP_IO.GPIO as GPIO  # pylint: disable=F0401
```
Comment (sky3d) [inline chip/gpio.py:11]: may be better to use `from CHIP_IO import GPIO` ?

File: chip/sensors.py [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+# Put your device token here. To get the token,
+# sign up at https://cloud4rpi.io and create a device.
+DEVICE_TOKEN = '__YOUR_DEVICE_TOKEN__'
+
+#  / GND |────────────> GND
+# | DATA |─────────┬──> LCD-D2
+#  \ VCC |─┬─[4k7]─┘
+#          └──────────> 5V
+#  DS18B20 (bottom view)
+
```
Comment (sky3d) [inline chip/sensors.py:20]: nice! 👍

File: common/minimal.py [hunk 4]
```diff
@@ -42,22 +42,22 @@ def osname():
 def main():
     # Put variable declarations here
     variables = {
-        'RoomTemp': {
+        'Room Temp': {
```
Comment (sky3d) [inline common/minimal.py:45]: Not so good approach. Variable name uses as 'id' in payloads, value calculation etc.  On UI of a web-site a user can to redefine names as human-readable strings.But in configuration and payloads names | id without space is more preferable.

File: omega2/omega2.py (hunk 1 above)
Comment (Himura2la) [reply omega2/omega2.py:39]: isinstance() is good when we need to catch a single type, but here the isinstance() usage will look much less elegant IMHO.
```
if isinstance(state, bool) or isinstance(state, int):
```
Yuck! I feel like writing in C...

File: common/minimal.py (hunk 4 above)
Comment (Himura2la) [reply common/minimal.py:45]: I see that it is a string. Why can't I use spaces in strings? What will break and in what cases? I don't think that saving one byte of data in each message worth worsening the readability and elegance of these messages.

File: chip/gpio.py (hunk 2 above)
Comment (Himura2la) [reply chip/gpio.py:11]: Maybe. I simply followed the examples here https://github.com/xtacocorex/CHIP_IO 
We should check that `from CHIP_IO import GPIO` works, I'm not absolutely sure that these statements are equivalent

File: omega2/omega2.py (hunk 1 above)
Comment (Himura2la) [reply omega2/omega2.py:39]: Maybe I can do it better, but it feels overcomplicated 
`if any({isinstance(state, t) for t in {bool, int}}):`
And long
The `if type(state) in` is classical... Why pylint does not like it?

File: chip/gpio.py (hunk 2 above)
Comment (sky3d) [reply chip/gpio.py:11]: http://docs.python-guide.org/en/latest/writing/structure/#modules

**Pull request merged**